DataProc/PPass/main_rec.C

```diff
@@ -48,12 +56,20 @@ void main_rec(const char *filename="raw.root", const char* options="")
     rec.SetCDBSnapshotMode("OCDB.root");
   }
   else {
+    AliCDBManager* man = AliCDBManager::Instance();
     // setup ocdb by custom (if any) or default settings
     if (gSystem->AccessPathName("OCDBconfig.C", kFileExists)==0) {
       gROOT->ProcessLine("OCDBconfig.C");
     }
+    else if (gSystem->AccessPathName("localOCDBaccessConfig.C", kFileExists)==0) {
```

Comment:
If the override OCDB macro is there, we load it. This does not affect Grid jobs.